com.unity.render-pipelines.high-definition/Runtime/Material/Lit/Lit.cs

```diff
@@ -197,6 +197,9 @@ protected void GetGBufferOptions(HDRenderPipelineAsset asset, out int gBufferCou
             supportShadowMask = asset.currentPlatformRenderPipelineSettings.supportShadowMask;
             supportLightLayers = asset.currentPlatformRenderPipelineSettings.supportLightLayers;
             gBufferCount = 4 + (supportShadowMask ? 1 : 0) + (supportLightLayers ? 1 : 0);
+#if ENABLE_VIRTUALTEXTURES
+            gBufferCount++;
```

Comment:
This is something we spent a lot of time thinking about. In the end we decided that is an "all or nothing" setting. So you can disable VT for the project, in which case it will be fully compiled out but if you enable it it's  always on. The two main reasons were: We build different data into the packaged game (and soon asset bundles) for VT so supporting both would mean shipping 2X the data. A toggle would also mean re-introducing a shader keyword. @UnityAljosha has also done some discovery and talked to stakeholders so he might have something extra do add?

Reply:
Let me finish the review of the PR and we will setup a meeting to discuss all this details. thanks

Reply:
That's right. We assume a project is authored for VT so we decided to make this a project setting and not a per player setting. For streaming VT, the stack node can fall back to regular texture samples but the content is probably prohibitively large.  If a user want to ship on an unsupported platform then he can toggle VT off in the project setting and see where things break. We can relax this but we did not want to commit to automatic fallback behavior per platform at this time.

That say, perhaps there are scenarios where some scenes are using VT (with their own set of content) and some don't. We are not anticipating this though.